### PR TITLE
Add MsTimer2 library

### PR DESCRIPTION
**Pull request by ncolomer**
I could have use the following download links http://www.pjrc.com/teensy/arduino_libraries/MsTimer2.zip but preferred download them directly from the Github project.
I have also some doubts about `version`, `frameworks` and `platforms` values, information are a bit sparse on the internet :)

### CODE REVIEW

**Comment by ivankravets** [comment]
Looks excellent! Why you don't want to use direct "GitHub" repo?

**Comment by ncolomer** [comment]
That's the opposite: I preferred rely on Github zipball (see downloadUrl value) because I'm more confident than with the link on pjrc.com ;)

**Comment by ivankravets** [comment]
I agree with, that GitHub is more reliable. So, I propose to link this library with GitHub repo, not with Archive. See my proposition:
```json
{
  "name": "MsTimer2",
  "keywords": "timer, callback",
  "description": "MsTimer2 is a small and very easy to use library to interface Timer2 with humans. It's called MsTimer2 because it \"hardcodes\" a resolution of 1 millisecond on timer2.",
  "repository":
  {
    "type": "git",
    "url": "https://github.com/PaulStoffregen/MsTimer2.git"
  },
  "authors": [{
    "name": "Javier Valencia",
    "email": "javiervalencia80@gmail.com",
    "url": "http://www.pjrc.com/teensy/td_libs_MsTimer2.html"
  }, {
    "name": "Paul Stoffregen",
    "url": "https://www.pjrc.com/about/",
    "maintainer": true
  }],
  "frameworks": [
    "arduino"
  ],
  "platforms": [
    "atmelavr",
    "teensy"
  ]
}
```

Do you have any objections? :blush:

**Comment by ncolomer** [comment]
Perfect! (I had not seen this `repository` parameter)
I just apply the changes to this PR (updated commit 859a718).

File: configs/mstimer2.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "authors": [{
+    "name": "Javier Valencia",
+    "email": "javiervalencia80@gmail.com",
+    "url": "http://www.pjrc.com/teensy/td_libs_MsTimer2.html"
+  }, {
+    "name": "Paul Stoffregen",
+    "url": "https://www.pjrc.com/about/",
+    "maintainer": true
+  }],
+  "examples": "examples/*/*.pde",
```
Comment (ivankravets) [inline configs/mstimer2.json]: You can remove this line #18. If library contains "examples" directory in the root of `src`, PlatformIO will automatically scan for the examples.

Comment (ncolomer) [inline configs/mstimer2.json]: Fixed in both platformio-libmirror/pull/4 and platformio-libmirror/pull/5 :)

**Comment by ivankravets** [comment]
Thanks :smile: 

I propose you to pull this request to Paul's repo. I use "platformio-libmirror" only for the libs which haven't own repository (located on forum, and etc.). If library has own repo, the I pull request to it.

See my the latest pull-requests to Paul's repos: https://github.com/ivankravets

To my mind, this is good idea to keep `library.json` within real source code of library because an author of library will have ability to edit/correct it.

**Comment by ncolomer** [comment]
Makes sense!

See freshly backed PRs:
* https://github.com/PaulStoffregen/MsTimer2/pull/2
* https://github.com/PaulStoffregen/FlexiTimer2/pull/1

**Comment by ivankravets** [comment]
Thanks! Now you can register these libs via `platformio`:
```shell
> platformio lib register %URL_TO_LIBRARY_JSON%
```

For example, go to https://github.com/PaulStoffregen/MsTimer2 , then `library.json`, then `RAW` button and copy this URL :smile: 

I will see these libs after registration and approve them.

**Comment by ncolomer** [comment]
Done!

**Comment by ivankravets** [comment]
Done too :+1: 

* http://platformio.org/#!/lib/show/137/MsTimer2
* http://platformio.org/#!/lib/show/138/FlexiTimer2

**Comment by ivankravets** [comment]
How about to add the popular libs form GitHub? 

I've just searched the bunch of interesting libs by query [arduino library](https://github.com/search?o=desc&q=arduino+library&s=stars&type=Repositories&utf8=✓)

I added more then 100 first libs from this list. Maybe, I missed something. You can use search on PlatformIO before create new library. :beers:

**Comment by ncolomer** [comment]
I'm thinking about a piece of script to check (and potentially import?) missing libraries automatically from [your](https://github.com/search?l=C%2B%2B&o=desc&q=arduino+library&s=stars&type=Repositories&utf8=%E2%9C%93) search (plus the C++ language filter, could do the same for C).

It should not be so difficult to achieve but it may be needed to have access to PlatformIO lib registry master data (or work on a dump). WDYT?

**Comment by ivankravets** [comment]
> WDYT?

I'm not sure that we can parse GitHub search result and generate proper `library.json`. The key feature of `library.json` are `name` and `description`. I find this data in Internet and fill with human-readble information for beginners. 

I agree, that we can generate a list with the potential libs, but I'm not sure that we should use them without any changes.

**Comment by ncolomer** [comment]
Close this since the lib was added, thanks!

**Pull request closed without merging**